sycl/source/detail/config.hpp

```diff
@@ -120,7 +120,7 @@ template <> class SYCLConfig<SYCL_BE> {
     const char *ValStr = BaseT::getRawValue();
     const std::array<std::pair<std::string, backend>, 3> SyclBeMap = {
         {{"PI_OPENCL", backend::opencl},
-         {"PI_LEVEL0", backend::level0},
+         {"PI_LEVEL0", backend::level_zero},
```

Comment:
```suggestion
         {"PI_LEVEL_ZERO", backend::level_zero},
```

Reply:
also docs which specify env variables need to be updated

Reply:
Done.